SOLUTIONS/helloworld.py

```diff
@@ -1,2 +1 @@
-# Print string to terminal
-print('Hello world!')
+print "Hello World"
```

Comment:
This change is incompatible with Python 3.

Reply:
I was trying in python 2. Didn't see there was one already with python 3

Reply:
The end of life of Python 2 is coming up quickly http://pythonclock.org so please try come up with a unique filename and try to write a solution that will work in Python 3.

Reply:
Thanks for the info, Will do it in py3 and commit the changes